src/analyzing/AST_to_IL.ml

```diff
@@ -1167,7 +1167,9 @@ and type_ env (ty : G.type_) : type_ =
 
 (* TODO: What other languages have no fallthrough? *)
```

Comment:
maybe study a bit more the other languages and remove this comment? or just remove the comment.